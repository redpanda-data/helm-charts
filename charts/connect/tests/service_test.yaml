suite: service tests
templates:
  - service.yaml
tests:
  - it: should enable service default
    asserts:
      - matchSnapshot:
          path: spec

  - it: should enable service custom
    set:
      service:
        type: LoadBalancer
        port: 8080
        extraPorts:
          - name: extraPort
            port: 9999
            targetPort: 9999
            protocol: UDP
    asserts:
      - matchSnapshot:
          path: spec
