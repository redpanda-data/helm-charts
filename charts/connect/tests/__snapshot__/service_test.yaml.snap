should enable service custom:
  1: |
    ports:
      - name: http
        port: 8080
        protocol: TCP
        targetPort: http
      - name: extraPort
        port: 9999
        protocol: UDP
        targetPort: 9999
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: redpanda-connect
    type: LoadBalancer
should enable service default:
  1: |
    ports:
      - name: http
        port: 80
        protocol: TCP
        targetPort: http
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: redpanda-connect
    type: ClusterIP
